api/internal/store/entity/schedule.go

```diff
@@ -0,0 +1,43 @@
+package entity
+
+import (
+	"time"
+
+	"github.com/and-period/furumaru/api/pkg/uuid"
+	"gorm.io/gorm"
+)
+
+// Schedule - 開催スケジュール
+type Schedule struct {
+	ID          string         `gorm:"primarykey;<-:create"` // テンプレートID
```

Comment:
```suggestion
	ID          string         `gorm:"primaryKey;<-:create"` // テンプレートID
```

https://gorm.io/docs/models.html